packages/core/test/lib/base.test.ts

```diff
@@ -887,7 +887,7 @@ describe('BaseClient', () => {
       expect(capturedEvent).toEqual(normalizedTransaction);
     });
 
-    test('calls beforeSend and uses original event without any changes', () => {
+    test('calls `beforeSend` and uses original event without any changes', () => {
```

Comment:
l: While re-writing these tests, an idea would also be to maybe nest them into `describe('beforeSend')` and `describe('beforeSendTransaction')`?

Reply:
I def think this test file could be streamlined/better organized more than I have here (some `describe.each()` or `it.each()` calls are probably in order), but for the sake of keeping it still relatively clear what changes are actually for `beforeSendTransaction` and which are for re-org purposes, IMHO that should happen in a separate PR. (I may actually pull the re-org changes here out into a separate PR as it is.)

Reply:
makes sense to me! 👍